packages/swing-store/src/bundleStore.js

```diff
@@ -162,8 +163,10 @@ export function makeBundleStore(db, ensureTxn, noteExport = () => {}) {
     return sqlGetPrunedBundles.all();
   }
 
-  function assertComplete(level) {
-    assert.equal(level, 'operational'); // for now
+  function assertComplete(artifactMode) {
+    if (artifactMode === 'debug') {
+      return; // no validation in this mode
```

Comment:
I think we can remove this check for debug?

Reply:
Well, only if we're saying that `debug` always does the same checks as `operational` everywhere. I changed the entrypoints to map both `debug` and `operational` to use `operational` on all the submodules. If we change the submodules to treat them the same way, then we should remove that entrypoint-side mapping.

I won't try to convince you that we should have a no-validation mode at all, so I'll remove the entrypoint guard and change the submodules to treat `debug` and `operational` as the same.

Reply:
after discussion on Slack, we decided to make `assertComplete` not accept `debug` (i.e. the "check modes" are a subset of `ArtifactMode`), and to continue having `importer.js` map `artifactMode === 'debug'` to a `checkMode = 'operational'`